spec/migrations/20160318133352_add_deprecated_to_vms.rb

```diff
@@ -0,0 +1,15 @@
+require_migration
```

Comment:
No need to test this since you are basically just testing Rails.  So, this file can just go away.